WordPress/Classes/ViewRelated/Plans/PlanListViewController.swift

```diff
@@ -109,26 +124,40 @@ struct PlanListViewModel {
             return navigationVC
         }
     }
-
-    // TODO: Prices should always come from StoreKit
-    // @koke 2016-02-02
-    private func priceForPlan(plan: Plan) -> String {
-        switch plan {
-        case .Free:
-            return ""
-        case .Premium:
-            return "$99.99"
-        case .Business:
-            return "$299.99"
-        }
-    }
 }
 
-final class PlanListViewController: UITableViewController, ImmuTablePresenter {
+final class PlanListViewController: UITableViewController, ImmuTablePresenter, WPNoResultsViewDelegate {
```

Comment:
Nitpicky, but could the `WPNoResultsViewDelegate` implementation either be in its own extension on the view controller, or otherwise the methods just grouped under a `// MARK:`?